### PR TITLE
Skysocks rename

### LINKED ISSUE #97: Rename therealproxy and skyproxy to skysocks
Should be done in:
- [x] `skywire-mainnet`
- [x] `skywire-services`

### PR DESCRIPTION
**Pull request by Kifen**
Fixes #97

 Changes:	
-	Changed all occurrence of therealproxy and skyproxy to skysocks

How to test this PR:

### CODE REVIEW

**Comment by jdknives** [comment]
@Kifen there are plenty of Milestone 2 commits that would get merged into master. We should not do that yet. This needs to be pointing to m2.

**Comment by Kifen** [comment]
@jdknives should I close the PR?

File: README.md [hunk 1]
```diff
@@ -185,7 +185,12 @@ After `skywire-visor` is up and running with default environment, default apps a
 
 - [Chat](/cmd/apps/skychat)
 - [Hello World](/cmd/apps/helloworld)
+<<<<<<< HEAD
```
Comment (nkryuchkov) [inline README.md]: Unresolved git conflict.

File: ci_scripts/run-internal-tests.sh [hunk 2]
```diff
@@ -17,4 +17,4 @@ go clean -testcache &>/dev/null || go test -race -tags no_ci -cover -timeout=5m
 go clean -testcache &>/dev/null || go test -race -tags no_ci -cover -timeout=5m github.com/SkycoinProject/skywire-mainnet/internal/noise -run TestReadWriterXKPattern >> ./logs/internal/TestReadWriterXKPattern.log
 go clean -testcache &>/dev/null || go test -race -tags no_ci -cover -timeout=5m github.com/SkycoinProject/skywire-mainnet/internal/noise -run TestReadWriterConcurrentTCP >> ./logs/internal/TestReadWriterConcurrentTCP.log
 
-go clean -testcache &>/dev/null || go test -race -tags no_ci -cover -timeout=5m github.com/SkycoinProject/skywire-mainnet/internal/therealproxy -run TestProxy >> ./logs/internal/TestProxy.log
+go clean -testcache &>/dev/null || go test -race -tags no_ci -cover -timeout=5m github.com/SkycoinProject/skywire-mainnet/internal/skysocks -run TestProxy >> ./logs/internal/TestProxy.log
```
Comment (nkryuchkov) [inline ci_scripts/run-internal-tests.sh]: Please add a newline at the end of the file.

File: docker/images/node/Dockerfile [hunk 3]
```diff
@@ -18,9 +18,9 @@ RUN go build -mod=vendor -tags netgo -ldflags="-w -s" \
     go build  -mod=vendor -ldflags="-w -s" -o skywire-cli ./cmd/skywire-cli	&&\
     go build  -mod=vendor -ldflags="-w -s" -o ./apps/skychat.v1.0 ./cmd/apps/skychat	&&\
 	  go build  -mod=vendor -ldflags="-w -s" -o ./apps/helloworld.v1.0 ./cmd/apps/helloworld &&\
-	  go build  -mod=vendor -ldflags="-w -s" -o ./apps/socksproxy.v1.0 ./cmd/apps/therealproxy &&\
-	  go build  -mod=vendor -ldflags="-w -s" -o ./apps/socksproxy-client.v1.0  ./cmd/apps/therealproxy-client
-
+	  go build  -mod=vendor -ldflags="-w -s" -o ./apps/socksproxy.v1.0 ./cmd/apps/skysocks &&\
+	  go build  -mod=vendor -ldflags="-w -s" -o ./apps/socksproxy-client.v1.0  ./cmd/apps/skysocks-client &&\
```
Comment (nkryuchkov) [inline docker/images/node/Dockerfile]: Why do we need `&&\` at the end of the line?

**Review by nkryuchkov — CHANGES_REQUESTED** [review]
There are still some `socksproxy` strings that should be renamed as well.

File: README.md (hunk 1 above)
Comment (Kifen) [reply README.md]: Resolved

File: ci_scripts/run-internal-tests.sh (hunk 2 above)
Comment (Kifen) [reply ci_scripts/run-internal-tests.sh]: New line has been added

File: docker/images/node/Dockerfile (hunk 3 above)
Comment (Kifen) [reply docker/images/node/Dockerfile]: No we don't; it's been removed.

**Review by nkryuchkov — CHANGES_REQUESTED** [review]
Looks good! However, `make check` is broken and needs to be fixed. Also, there's a merge conflict. I think we can merge after those are resolved.

**Review by nkryuchkov — APPROVED** [review]

**Pull request merged**